torchaudio/pipelines/_wav2vec2/impl.py

```diff
@@ -1177,3 +1180,90 @@ def _get_state_dict(self, dl_kwargs):
 
 Please refer to :py:class:`torchaudio.pipelines.Wav2Vec2ASRBundle` for the usage.
 """  # noqa: E501
+
+
+WAVLM_BASE = Wav2Vec2Bundle(
```

Comment:
could you add another pipeline named `WAVLM_BASE_PLUS`? It has the same model architecture but is trained on more data, the checkpoint is in https://github.com/microsoft/UniSpeech/tree/main/WavLM